services/plans/plan_templates/schemas.go

```diff
@@ -0,0 +1,211 @@
+package plan_templates
+
+import (
+	"regexp"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+	"github.com/leanspace/terraform-provider-leanspace/helper/general_objects"
+)
+
+var nameRegex = regexp.MustCompile(`^[a-zA-Z0-9_-]*$`)
+
+var planTemplateSchema = map[string]*schema.Schema{
+	"id": {
+		Type:     schema.TypeString,
+		Computed: true,
+	},
+	"asset_id": {
+		Type:         schema.TypeString,
+		Required:     true,
+		ValidateFunc: validation.IsUUID,
```

Comment:
it's missing `ForceNew: true` I think

Reply:
what's ForceNew again?

Reply:
`ForceNew` will force the recreation of the resource if this field changes (DELETE and CREATE instead of UPDATE)

Reply:
hmm we might miss it in a few other places